example/spec/parallel/queueGranules/queueGranulesPostProcessingSpec.js

```diff
@@ -208,8 +208,20 @@ describe('The Queue Granules workflow triggered with a database-schema-compliant
           { prefix: config.stackName, granuleId: granule.granuleId },
           'completed'
         );
+
+        const executionSearchResult = await searchExecutionsByGranules({
```

Comment:
This needed to be updated as the psql->api translation returns only one execution for the granule, and order isn't guaranteed both in write (out of order situations) and return from the database.    We are concerned here that the granule is complete and the execution is correctly assigned to the granule.